docs/src/manual/abstraction-based-control.md

```diff
@@ -0,0 +1,13 @@
+# Abstraction-based control
+
+Given a mathematical description of the system dynamics and the specifications describing the desired closed-loop behavior of the system, 
+**abstraction-based control** techniques involve synthesizing a **correct-by-construction** controller through a **systematic** three-step procedure. 
+First, both the original system and the specifications are transposed into an abstract domain, resulting in an abstract system and corresponding abstract specifications. 
+This step generally involves a complete discretization of the state and input spaces, typically with uniform hyperrectangles.
+Next, an abstract controller is synthesized to solve this abstract control problem. Finally, the third step, referred to as the **refinement procedure**, involves deducing a controller for the original control problem from the abstract controller. The value of this approach lies in the substitution of the original system (often an infinite system) with a finite system, which enables it to leverage powerful control tools in the domain of symbolic control. This three steps procedure is illustrated on the following figure.
+
```

Comment:
We could put the image here, and not as a link?

Same, we could just put it in the assets and then use 
```<img src="assets/abstraction.png" height="240" alt="Abstraction-based control">```

Reply:
I will try the local reference path